package.json

```diff
@@ -6,6 +6,8 @@
     "@lerna-lite/changed": "2.1.0",
     "@lerna-lite/cli": "2.1.0",
     "@lerna-lite/run": "2.1.0",
+    "@lerna-lite/version": "^2.1.0",
+    "@lerna/publish": "^6.6.1",
```

Comment:
Are you sure about non-exact versions (`^`)?

Reply:
good call 👌